test/pom.xml

```diff
@@ -53,7 +53,7 @@ THE SOFTWARE.
     <dependency>
       <groupId>${project.groupId}</groupId>
       <artifactId>jenkins-test-harness</artifactId>
-      <version>2.20</version>
+      <version>2.23</version>
```

Comment:
It is, to catch https://github.com/jenkinsci/jenkins-test-harness/pull/62 which indeed got [released as 2.23](https://github.com/jenkinsci/jenkins-test-harness/commit/f9c67d5527bd1ef7cf7b6139108f9eb200ee5c3a).